content/swiftui/concepts/viewmodifier/terms/textFieldStyle/textFieldStyle.md

```diff
@@ -0,0 +1,114 @@
+---
+Title: '.textFieldStyle()'
+Description: 'Applies a specified style to a TextField'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+  - 'Xcode'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.textFieldStyle()`** modifier applies a specified style to a [`TextField`](https://www.codecademy.com/resources/docs/swiftui/views/textfield).
+
+## Syntax
+
+```pseudo
+        TextField("label", text: $bindingValue)
+            .textFieldStyle(style)
+```
+
+The style passed as an argument to `.textFieldStyle()` is applied to the `TextField`. Built-in styles are available in Swift UI and it's also possible to create custom styles.
```

Comment:
Is it possible to list at least some of the built-in styles with descriptions?

Reply:
@SSwiniarski 
Thank you for reviewing and for the suggestions!
I added in some of the built-in styles with descriptions.